src/main/resources/application.yaml

```diff
@@ -75,3 +75,76 @@ uk:
             EXAMPLE: 2e00903c-46f4-46d6-9d94-d419ee30b927
           welsh:
             EXAMPLE: example-template-id-222
+
+notify:
+  api-key: fake-key
+  baseUrl: https://api.notifications.service.gov.uk
+
+payments:
+  api:
+    url: ${PAY_URL:}
+    callback-url: ${PAY_CALLBACK_URL:}
+
+fees-register:
+  api:
+    url: http://fees-register-api-demo.service.core-compute-demo.internal
+  parameters:
+    C100_SUBMISSION_FEE:
+      channel: default
+      event: miscellaneous
+      jurisdiction1: family
+      jurisdiction2: family court
+      keyword: ChildArrangement
+      service: private law
+
+# NEW BELOW THIS LINE
+
+
+idam:
+  s2s-auth:
+    url: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+    microservice: prl_cos_api
+    totp-secret: GT2V6PLQ34XGPTPM
+  api:
+    url: https://idam-api.aat.platform.hmcts.net
+  client:
+    id: prl-cos-api
+    redirect_uri: https://manage-case.aat.platform.hmcts.net/oauth2/callback
+    secret: W7supXKMJgcEWKBS
```

Comment:
secrets need to replaced

Reply:
we will sort this once we have secrets pulled from azure